draft-ietf-tls-tls13.md

```diff
@@ -918,7 +918,9 @@ Initial Handshake:
 Subsequent Handshake:
        ClientHello
          + pre_shared_key
-         + key_share*            -------->
+         + key_share*
+         + hello_finished*
```

Comment:
Bike shed (hence here and not on the list):

If we are to avoid confusion about this, could we call this a MAC rather than Finished.  Maybe `client_hello_mac`.  This isn't the end, so Finished is only a product of the (convoluted) process we used to reach this point.